dali/pipeline/init.cc

```diff
@@ -40,13 +43,30 @@ void subscribe_signals() {
 
 #endif
 
+void InitializeBufferPolicies() {
+  if (const char *threshold_str = std::getenv("DALI_HOST_BUFFER_SHRINK_THRESHOLD")) {
```

Comment:
Maybe we can extract this setter to some common place as now we have ranges spread in two places.

Reply:
I moved the max growth factor to buffer.h.